boards/xtensa/esp32_devkitc_wroom/esp32_devkitc_wroom.dts

```diff
@@ -29,6 +30,20 @@
 		};
 	};
 
+	esp-touch-sensor0 {
```

Comment:
shouldn't this be `&touch {`... and at root level now?

Reply:
I don't think so because esp-touch-sensor0 is not refering to /soc/touch. esp-touch-sensor0 (compatible = "espressif, touch-sensor') works the same way  buttons (compatible = "gipo-keys") seen some lines before. To be fair now I think this driver should have been named and refered as "espressif,touch-keys" instead of "espressif,touch-sensor". I think "espressif,touch-sensor" even under zephyr input subsystem (zephyr/driver/input/input_esp32_touch_sensor.c) is causing confusion.

Reply:
But should it not refer to `/soc/touch`? Right now on the final tere you have:

```
/ {
        soc {
                touch: touch@3ff48858 {
                        compatible = "espressif,touch-sensor";
                        reg = < 0x3ff48858 0x38 >;
                        interrupts = < 0x2e >;
                        interrupt-parent = < &intc >;
                        status = "disabled";
                };
        };
        esp-touch-sensor0 {
                compatible = "espressif,touch-sensor";
                status = "okay"; 
                ...
                touch_sensor_set: touch_sensor_0 {
                        channel-num = < 0x9 >;
                        channel-sens = < 0x14 >;
                        zephyr,code = < 0xb >;
                };
        };
};
```

I think this wants to be

```
/ {
        soc {
                touch: touch@3ff48858 {
                        compatible = "espressif,touch-sensor";
                        reg = < 0x3ff48858 0x38 >;
                        interrupts = < 0x2e >;
                        interrupt-parent = < &intc >;
                        status = "okay";
                        ...
                        touch_sensor_set: touch_sensor_0 {
                                channel-num = < 0x9 >;
                                channel-sens = < 0x14 >;
                                zephyr,code = < 0xb >;
                        };
                };
        };
};
```

which you would get by doing

```
&touch {
        status = "okay";
        touch_sensor_set: touch_sensor_0 {
                channel-num = < 0x9 >;
                channel-sens = < 0x14 >;
                zephyr,code = < 0xb >;
        };      
}
```

in the board file

Reply:
I'll make the necessary adjustments.